src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,47 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.enums.Message;
+import duke.exception.DukeException;
+import duke.ui.Ui;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+/**
+ * Adds Tasks to the TaskList and invokes appropriate UI messages about it
+ */
+public class AddCommand implements Command {
+    
+    private final String[] parsedInput;
+    private final String commandTag; // indicates what kind of Task
```

Comment:
Maybe it would be better if this comment was phrased according to the Javadocs standards, eg.

```
/** Indicates the type of task */
private final String commandTag;
```

Reply:
Yeap, will change it. Thanks :)